.env

```diff
@@ -1,5 +0,0 @@
-CMS_IMAGE=taccwma/protx-cms
```

Comment:
We do create that information and store it: https://github.com/TACC/Core-Portal-Deployments/blob/main/protx/camino/pprd.env

A dev env would require creating the `.env` file (could be just copy the `pprd.env`) and setting it to whatever specific images/tags they want for your local dev environment. We don't track this in the core portals for the same reason, so we can mix and match as we see fit in local dev. Otherwise it becomes a git change that you have to deal with, and becomes annoying when testing image builds and experimenting (or more commonly it gets changed for dev convenience then has to be remembered to get reset before merging back in). Either way its mental overhead we dont need and constrains flexibilty imo.